src/Servers/Kestrel/Core/src/Internal/Infrastructure/StringUtilities.cs

```diff
@@ -115,6 +117,111 @@ public static unsafe bool TryGetAsciiString(byte* input, char* output, int count
             return isValid;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
+        public static unsafe bool TryGetLatin1String(byte* input, char* output, int count)
```

Comment:
Why are we using unsafe code here instead of Span<byte> and Span<char>? To match TryGetAsciiString?

Reply:
Yeah. I'm trying to keep this as close to TryGetAsciiString as possible since it's conceptually quite similar.